virttest/utils_vdpa.py

```diff
@@ -261,3 +266,23 @@ def get_vdpa_pci(driver='mlx5_core'):
     for pci_info in pf_info.values():
         if pci_info.get("driver", "") == driver:
             return pci_info.get('pci_id')
+
+
+def get_dev_file_by_name(name):
```

Comment:
Suggest adding vdpa string to name it.
```suggestion
def get_vdpa_path_by_name(name):
```

Reply:
Support your point of view, it has been changed, thank you very much.